knock/src/core/ui/Toast/Toast.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import styles from '../../styles/toast.module.scss';
+
+const Toast: React.FC = () => {
+  const [alert, alertSet] = useState(true);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      alertSet(false);
+    }, 2000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  return (
+    <>
+      {alert === true ? (
+        <div className={`${styles['layout']}`}>
+          <div className={`${styles['text']}`}>URL이 복사되었습니다</div>
+        </div>
```

Comment:
URL이 복사되었습니다 부분은 동적으로 할당하는게 UX 적인 관점에서 더 효과적인 것 같은데 어떻게 생각하시나요??
```suggestion
        <div className={`${styles['layout']}`}>
          <div className={`${styles['text']}`}>{토스트_메세지}</div>
        </div>
```